HW2/04-background.html

```diff
@@ -14,7 +14,7 @@
       <div class = "mx-auto w-50 mt-5 bg-light pt-5 pb-5 p-lg-2">
       <h1>Exercise 04 - Background</h1>
       <p>
-        The background color will change every few seconds (as indicated below).
+        The background color will change every few seconds (as indicated below) .
         You can stop this behavior and you can also change how often the
         background color changes.
       </p>
```

Comment:
One lines 22 and 23 there is a WAVE audit error. your labels "for" attribute should have the same value as the inputs "id" attribute